databroker/broker.py

```diff
@@ -565,6 +565,26 @@ def process(self, headers, func, fields=None, fill=False):
         _process(mds=self.mds, fs=self.fs, headers=headers, func=func,
                  fields=fields, fill=fill)
 
+    def export(headers, mds, fs):
+        """ export a list of headers
+
+            Parameters:
+            -----------
+            headers : databroker.header
+                list of headers that are going to be exported
+            mds : metadatasote
```

Comment:
metadatasote -> MDS
metadata store object -> target instance of metadatastore to export into